internals/cli/cli_test.go

```diff
@@ -136,7 +136,7 @@ func (s *PebbleSuite) TestErrorResult(c *C) {
 	restore := fakeArgs("pebble", "warnings")
 	defer restore()
 
-	err := cli.RunMain()
+	err := cli.Run(cli.ClientConfig)
```

Comment:
Why not just `cli.Run(&client.Config{})` for this (and below)?

Reply:
That's fair (and actually the way I implemented it at first--only to realize there's no trivial way to get rid of the `cli.ClientConfig` global, so I used it anyway).